pyomo/contrib/pynumero/tests/test_cyipopt_examples.py

```diff
@@ -36,13 +45,13 @@ def test_status_maps(self):
 
 class TestExamples(unittest.TestCase):
     def test_external_grey_box_react_example_maximize_cb_outputs(self):
-        ex = import_file(os.path.join(example_dir, 'external_grey_box', 'react-example', 'maximize_cb_outputs.py'))
+        ex = import_file(os.path.join(example_dir, 'external_grey_box', 'react_example', 'maximize_cb_outputs.py'))
```

Comment:
Does `import_file()` not work if there are "`-`" in the filenames / paths?

Reply:
The issue here wasn't `import_file` not working - it was a relative import at the top of the file that was causing issues. `import_file` doesn't care.